### PR TITLE
Fix unused arguments

### PR DESCRIPTION
**Pull request by Andrei-Aksionov**
Hi there 👋 

Another “chores” PR of mine.

This time I want to tackle the unused arguments.
Basically the list with changes one can find in commit messages.

Now the CLI command looks like this (added `ARG001` rule):
```bash
ruff check * --select E,W,F,S --extend-select C4,SIM,RET,PT,I001,ANN001,ANN201,ANN205,ANN206,ARG001 --ignore E501,E731,S108,S101,S113,S603,PT007,S310,E402,PT004,C408 --per-file-ignores "*/**/__init__.py":I001,"tests/*":ANN --line-length 120 --target-version py38 --fix; black -l120 -C --preview --target-version py38 chat eval finetune generate lit_gpt notebooks pretrain quantize scripts tests xla
```

### CODE REVIEW

File: finetune/lora.py [hunk 1]
```diff
@@ -93,10 +93,10 @@ def setup(
     logger = CSVLogger(out_dir.parent, out_dir.name, flush_logs_every_n_steps=log_interval)
     fabric = L.Fabric(devices=devices, strategy=strategy, precision=precision, loggers=logger, plugins=plugins)
     fabric.print(hparams)
-    fabric.launch(main, data_dir, checkpoint_dir, out_dir, quantize)
```
Comment (Andrei-Aksionov) [inline finetune/lora.py:96]: This is the leftover from [previous](https://github.com/Lightning-AI/lit-gpt/blob/cc190152085ac60fe708455b2df8877f82d37087/finetune/lora.py#L88) quantization approach.

File: tests/test_model.py [hunk 2]
```diff
@@ -43,7 +43,15 @@ def test_against_gpt_neox_model(rotary_pct, batch_size, n_embd, parallel_residua
     from scripts.convert_hf_checkpoint import copy_weights_gpt_neox
 
     batch_size = 3
-    ours_config = Config(block_size=64, vocab_size=100, n_layer=4, n_head=8, n_embd=n_embd)
+    ours_config = Config(
```
Comment (Andrei-Aksionov) [inline tests/test_model.py:46]: Before `rotary_pct` and `parallel_residual` arguments were provided, but not used.

**Review by awaelchli — APPROVED** [review]

File: lit_gpt/adapter.py [hunk 3]
```diff
@@ -161,5 +161,5 @@ def mark_only_adapter_as_trainable(model: GPT) -> None:
         param.requires_grad = adapter_filter(name, param)
 
 
-def adapter_filter(key: str, value: Any) -> bool:
+def adapter_filter(key: str, value: Any) -> bool:  # noqa: ARG001
```
[the remark this answers is not in the document]
Comment (Andrei-Aksionov) [reply lit_gpt/adapter.py]: I was hesitant about that. “Half of me” thought that it was okay in this case, the other “half” didn't.
Now, with your vote, it's 1.5 vs. 0.5 in favor of not adding it 😆.

Fixed.

File: tests/test_model.py (hunk 2 above)
Comment (carmocca) [reply tests/test_model.py:46]: This is also fixed in https://github.com/Lightning-AI/lit-gpt/pull/606

**Review by carmocca — APPROVED** [review]
Can you also include the `xla` directory in your command?

**Comment by Andrei-Aksionov** [comment]
> Can you also include the xla directory in your command?

Yeah, didn't notice that `black` isn't applied on `xla` folder.

```bash
ruff check * --select E,W,F,S --extend-select C4,SIM,RET,PT,I001,ANN001,ANN201,ANN205,ANN206,ARG001 --ignore E501,E731,S108,S101,S113,S603,PT007,S310,E402,PT004,C408 --per-file-ignores "*/**/__init__.py":I001,"tests/*":ANN --line-length 120 --target-version py38 --fix; black -l120 -C --preview --target-version py38 chat eval finetune generate lit_gpt notebooks pretrain quantize scripts tests xla
```

(Also updated the command in the PR message).

File: tests/test_model.py (hunk 2 above)
Comment (Andrei-Aksionov) [reply tests/test_model.py:46]: Oh, I see.
Should I then remove these changes from my PR, since yours is technically was first to change it? 😄

Comment (carmocca) [reply tests/test_model.py:46]: No, it's okay. I just wanted to mention that this wasn't forgotten

**Pull request merged**